Tools/i18n/msgfmt.py

```diff
@@ -98,18 +105,43 @@ def generate():
     return output
 
 
-def make(filename, outfile):
-    ID = 1
-    STR = 2
-    CTXT = 3
+def make(filenames, outfile):
+    """This function is now member of the public interface.
```

Comment:
```suggestion
    """
```
I wouldn't say its necessary and but it should not be the first line.